<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlantScan3D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
	border-right: 1px solid rgb(171, 171, 171);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="MainViewer" name="viewer" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExportView">
     <property name="title">
      <string>Export View</string>
     </property>
     <addaction name="actionSaveSnapshot"/>
    </widget>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionExportViewBefore"/>
    <addaction name="menuExportView"/>
    <addaction name="separator"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionWhiteTheme"/>
     <addaction name="actionBlackTheme"/>
    </widget>
    <addaction name="actionThemeBefore"/>
    <addaction name="menuTheme"/>
    <addaction name="separator"/>
    <addaction name="actionShowLeftSidebar"/>
    <addaction name="actionShowRightSidebar"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSaveSnapshot">
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionWhiteTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionBlackTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionExportViewBefore">
   <property name="text">
    <string>Export View Before</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionThemeBefore">
   <property name="text">
    <string>Theme Before</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionShowLeftSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Left Sidebar</string>
   </property>
   <property name="shortcut">
    <string>Alt+0</string>
   </property>
  </action>
  <action name="actionShowRightSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Right Sidebar</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainViewer</class>
   <extends>QWidget</extends>
   <header>.main_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
